platform.md

```diff
@@ -353,50 +359,66 @@ Usage:
+| `<plan>`       | `CNB_PLAN_PATH`      | `<layers>/plan.toml`                                   | Path to output resolved build plan                                                                                                                           |
+| `<platform>`   | `CNB_PLATFORM_DIR`   | `/platform`                                            | Path to platform directory                                                                                                                                   |
 
 ##### Outputs
-| Output             | Description
-|--------------------|----------------------------------------------
-| [exit status]      | (see Exit Code table below for values)
-| `/dev/stdout`      | Logs (info)
-| `/dev/stderr`      | Logs (warnings, errors)
-| `<group>`          | Detected buildpack group  (see [`group.toml`](#grouptoml-toml))
-| `<plan>`           | Resolved Build Plan (see [`plan.toml`](#plantoml-toml))
-
-| Exit Code | Result|
-|-----------|-------|
-| `0`       | Success
-| `11`      | Platform API incompatibility error
-| `12`      | Buildpack API incompatibility error
-| `1-10`, `13-19` | Generic lifecycle errors
-| `20`     | All buildpacks groups have failed to detect w/o error
-| `21`     | All buildpack groups have failed to detect and at least one buildpack has errored
-| `22-29` | Detection-specific lifecycle errors
+| Output                                                   | Description                                                                                              |
+|----------------------------------------------------------|----------------------------------------------------------------------------------------------------------|
+| [exit status]                                            | (see Exit Code table below for values)                                                                   |
+| `/dev/stdout`                                            | Logs (info)                                                                                              |
+| `/dev/stderr`                                            | Logs (warnings, errors)                                                                                  |
+| `<group>`                                                | Detected buildpack group  (see [`group.toml`](#grouptoml-toml))                                          |
+| `<plan>`                                                 | Resolved Build Plan (see [`plan.toml`](#plantoml-toml))                                                  |
+| `<analyzed>`                                             | Updated to include the run image obtained from applying generated Dockerfiles                            |
+| `<output>/generated/run/<image extension ID>/Dockerfile` | Generated Dockerfiles (see [Image Extension Specfication](image-extension.md)                            |
```

Comment:
I just want to confirm, we don't include `<image extension version>` in the path because we assume only one version of any given extension is valid. Is that right? Do we spell that out anywhere?

Reply:
This is a side effect of the way detect works (only one version of an image extension or buildpack can detect). I don't even know if this is spelled out clearly for buildpacks, but it can be seen here: https://github.com/buildpacks/spec/blob/main/platform.md#inputs-7

Reply:
(note the `<layers>/<buildpack-id>/<layer>/`)

Reply:
Is there anyway to add more clarification to the spec so that it's less of an inferred condition and more of a requirement?